code/game/machinery/doors/airlock.dm

```diff
@@ -1407,6 +1407,8 @@
 		A.previous_assembly = previous_airlock
 		A.update_name()
 		A.update_icon()
+		//NSV13 - I'm lazy okay?
+		A.nsv_ify(src)
```

Comment:
Did you just add a proc to convert a vanilla airlock to our airlocks?
![Deadpan outlined](https://user-images.githubusercontent.com/43185112/114013344-b7584180-9867-11eb-986a-50c765ae45a5.png)

Reply:
:)))

Reply:
What if someone actually wants an old type of airlock?

Reply:
It has a type check to only apply to ours